weather_mv/loader_pipeline/pipeline.py

```diff
@@ -66,7 +76,8 @@ def __open_dataset_file(filename: str) -> xr.Dataset:
             raise
     # Trying with explicit engine for cfgrib.
     try:
-        return xr.open_dataset(filename, engine='cfgrib', backend_kwargs={'indexpath': ''})
+        return _make_grib_dataset_inmem(
```

Comment:
Can you add a flag so the user can control if the dataset should be opened in memory? I can see this tool being used on O(100GB) files, where users would like to turn this off.

Reply:
I'm open to creating an issue to address this in the future.

Reply:
Agreed, let's open an issue and get this submitted for now.